tests/ui/asm/x86_64/issue-96797.rs

```diff
@@ -1,5 +1,5 @@
 // build-pass
-// compile-flags: -O
+//@compile-flags: -O
 // min-llvm-version: 14.0.5
```

Comment:
fun. Not sure how to do this yet

Reply:
Well you're in luck, most of these got removed :D

(though yes it's still a general issue for migration)